core-common/src/main/resources/event-type-schema/_metadata/_metadata.3.avsc

```diff
@@ -0,0 +1,97 @@
+{
+   "name":"Metadata",
+   "type":"record",
+   "namespace":"org.zalando.nakadi.generated.avro",
+   "doc":"Event metadata defines data is not related to actual event payload data",
+   "fields":[
+      {
+         "name":"occurred_at",
+         "type":{
+            "type":"long",
+            "logicalType":"timestamp-millis"
+         }
+      },
+      {
+         "name":"eid",
+         "type":{
+            "type":"string",
+            "logicalType":"uuid"
+         }
+      },
+      {
+         "name":"flow_id",
+         "type":[
+            "null",
+            "string"
+         ],
+         "default":null
+      },
+      {
+         "name":"received_at",
+         "type":[
+            "null",
+            {
+               "type":"long",
+               "logicalType":"timestamp-millis"
+            }
+         ],
+         "default":null
+      },
+      {
+         "name":"schema_version",
+         "type":"string"
+      },
+      {
+         "name":"published_by",
```

Comment:
I think this should also be nullable, as it's set by Nakadi, not by the clients.